backend/FwLite/LcmCrdt/Changes/Entries/RemovePublicationChange.cs

```diff
@@ -0,0 +1,15 @@
+using SIL.Harmony.Changes;
+using SIL.Harmony.Core;
+using SIL.Harmony.Entities;
+
+namespace LcmCrdt.Changes.Entries;
+
+public class RemovePublicationChange(Guid entityId, Guid publicationId) : EditChange<Entry>(entityId), ISelfNamedType<RemovePublicationChange>
+{
+    public Guid PublicationId { get; } = publicationId;
+    public override ValueTask ApplyChange(Entry entity, IChangeContext context)
+    {
+        entity.PublishIn = entity.PublishIn.Where(t => t.Id != PublicationId).ToList();
```

Comment:
In the replace change you're doing this, which is presumably slightly more efficient
```suggestion
        entity.PublishIn.RemoveAll(t => t.Id == PublicationId);
```

Reply:
Done in commit 55fb3ffd4; thanks for the suggestion.